### PR TITLE
add help message in doc

### PR DESCRIPTION
**Pull request by hhhowyou**
## What have you changed? (mandatory)

I met this `build failed` error in my usage, so I add it to the doc. Maybe it will help others.

Add help message in `doc/client/go-client-api.md`.

### CODE REVIEW

File: docs/clients/go-client-api.md [hunk 1]
```diff
@@ -124,6 +124,16 @@ found val:  for key: Company
 
 RawKVClient is a client of the TiKV server and only supports the GET/PUT/DELETE/SCAN commands. The RawKVClient can be safely and concurrently accessed by multiple goroutines, as long as it is not closed. Therefore, for one process, one client is enough generally.
 
+### Possiable Error
+
+If you get the error like follows.
```
Comment (siddontang) [inline docs/clients/go-client-api.md]: em, I think we should use update the doc to use tikv client directly

@disksing

File: docs/clients/go-client-api.md [hunk 2]
```diff
@@ -124,6 +124,16 @@ found val:  for key: Company
 
 RawKVClient is a client of the TiKV server and only supports the GET/PUT/DELETE/SCAN commands. The RawKVClient can be safely and concurrently accessed by multiple goroutines, as long as it is not closed. Therefore, for one process, one client is enough generally.
 
+### Possiable Error
```
Comment (ice1000) [inline docs/clients/go-client-api.md]: s/Possiable/Possible/

File: docs/clients/go-client-api.md [hunk 3]
```diff
@@ -124,6 +124,16 @@ found val:  for key: Company
...
+
+If you get the error like follows.
+
+```bash
+build rawkv-demo: cannot load github.com/pingcap/pd/pd-client: cannot find module providing package github.com/pingcap/pd/pd-client
+```
+
+You can run the `GO111MODULE=on go get -u github.com/pingcap/tidb@master` to fix it.
```
Comment (ice1000) [inline docs/clients/go-client-api.md]: s/the//

**Comment by ice1000** [comment]
Missing sign-off in the most recent commit

**Comment by hhhowyou** [comment]
I sign off the wrong commit. I will start a new pr...

**Pull request closed without merging**